go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/authzed/spicedb
 
-go 1.22.4
```

Comment:
I don't understand why go mod keeps updating those. And why even the minor version is relevant here. This used to mean "the go version you need to compile this project". Has anything changed in the go toolchain? It's annoying that this keeps bumping

Reply:
Yes, it now means the go version to use and `toolchain` is the toolchain to use

Reply:
Changed back since it broke WASM